libr/core/linux_heap_glibc.c

```diff
@@ -21,6 +21,33 @@
 #define GHT_MAX UT64_MAX
 #endif
 
+static bool GH(is_tcache)(RCore *core) {
+	char *fp;
+	double v;
+	if (r_config_get_i (core->config, "cfg.debug")) {
+		RListIter *iter;
+		RDebugMap *map;
+		r_debug_map_sync (core->dbg);
+		r_list_foreach (core->dbg->maps, iter, map) {
```

Comment:
this loop is dupped. just set an RList = core->dbg->maps, and core->io->maps

Reply:
but one is RDebugMap and the other RIOMap